views/2023/social-events.haml

```diff
@@ -0,0 +1,36 @@
+.introduction
+  .container-full.mb-4
+    %h3.event-title An array of fun and varied activities to surround the conference!
+
+.events
+  .container-full
+    .row.mb-5
+      .col-lg-3.col-md-4
+        .avatar.social-event-avatar
+          %img{ src: '/2023/images/alto.jpg' }
```

Comment:
Should this image be `kingpin.jpg`?

Reply:
Yes, good catch thank you. I mustn't have committed it in the last round.